packages/web/docs/src/pages/docs/other-integrations/ci-cd.mdx

```diff
@@ -71,16 +71,15 @@ jobs:
   publish:
     runs-on: ubuntu-22.04
     steps:
-      - name: checkout
-        uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: schema publish
         env:
-          HIVE_TOKEN: ${{ secrets.HIVE_TOKEN }}
```

Comment:
we should still keep the `HIVE_TOKEN`

Reply:
I guess what you changed would work but I have always used environment and that's how I have seen most docs

Reply:
I tried it in my company's CI/CD, and it worked fine without the `$HIVE_TOKEN` environment variable. Is there a specific reason why it's needed? Aren't we already passing it as a command line argument anyway?